llvm/lib/Transforms/IPO/DynUntrustedAllocPre.cpp

```diff
@@ -251,13 +261,17 @@ class DynUntrustedAllocPre : public ModulePass {
     auto rust_dealloc = M.getFunction("__rust_dealloc");
 
     auto rust_untrusted_alloc = M.getFunction("__rust_untrusted_alloc");
+    assert(rust_untrusted_alloc != nullptr && "Missing rust_untrusted_alloc.");
     auto rust_untrusted_alloc_zeroed =
         M.getFunction("__rust_untrusted_alloc_zeroed");
+    //assert(rust_untrusted_alloc_zeroed != nullptr && "Missing rust_untrusted_alloc_zeroed.");
     rust_untrusted_alloc->setLinkage(
         llvm::GlobalValue::LinkageTypes::ExternalLinkage);
-    rust_untrusted_alloc_zeroed->setLinkage(
+    //if (rust_untrusted_alloc_zeroed) {
```

Comment:
why remove the pointer check? seems like the assert is gone, so you may need it, right?

Reply:
Re-added pointer check.